website/homepage/src/guides/__init__.py

```diff
@@ -153,18 +153,6 @@ def build_guides(output_dir, jinja_env):
             category_file = os.path.join(output_folder, "index.html")
             with open(category_file, "w") as index_html:
                 index_html.write(output)
-        if guide["name"] == "01_quickstart":
-            output_folder = os.path.join(output_dir, "getting_started")
-            os.makedirs(output_folder)
-            index_file = os.path.join(output_folder, "index.html")
-            with open(index_file, "w") as index_html:
-                index_html.write(output)
-            output_folder = os.path.join(output_dir, "quickstart")
-            os.makedirs(output_folder)
-            index_file = os.path.join(output_folder, "index.html")
-            with open(index_file, "w") as index_html:
-                index_html.write(output)
```

Comment:
What were these files for before?

Reply:
when we changed `/getting_started` to be `/quickstart` I wrote this so that the old links to /getting_started would still work. not necessary now because I redirect them on nginx. which is much better for seo because its an actual redirect to a single page instead of this which was creating too separate pages with the same content.